### PR TITLE
Update eslint configuration to a more standard one

### PR DESCRIPTION
**Pull request by tusbar**
```
{
  "extends": [
    "standard",
    "standard-react"
  ],
  "plugins": [
    "jsx-a11y"
  ],
  "globals": {
    "__DEV__": false,
    "__TEST__": false,
    "__PROD__": false
  },
  "rules": {
    "comma-dangle": "off",
    "indent": "warn",
    "jsx-quotes": [2, "prefer-single"],
    "key-spacing": "off",
    "no-console": "warn",
    "no-debugger": "error",
    "no-irregular-whitespace": "off",
    "object-curly-spacing": [2, "always"],
    "react/prop-types": "warn",
    "space-before-function-paren": "off"
  }
}
```

This can always change, but I believe it is a good start.

### CODE REVIEW

**Review by jdesboeufs — APPROVED** [review]

File: .eslintrc [hunk 1]
```diff
@@ -1,32 +1,51 @@
...
-        "error",
-        "single"
+  "parser": "babel-eslint",
+  "env": {
+    "browser": true
+  },
+  "extends": [
+    "standard",
+    "standard-react"
+    // "plugin:jsx-a11y/strict"
```
Comment (tmerlier) [inline .eslintrc:9]: Commentaire oublié ?

Comment (tusbar) [reply .eslintrc:9]: no https://github.com/etalab/inspire/pull/418/commits/9f86a2e02b7882ec9585797794b52b650d32d329 :)

File: src/components/Producers/OtherProducersItem.js [hunk 2]
```diff
@@ -1,10 +1,29 @@
 import React from 'react'
+import PropTypes from 'prop-types'
 import { find } from 'lodash'
+
 import { item } from './OtherProducersItem.scss'
```
Comment (tmerlier) [inline src/components/Producers/OtherProducersItem.js]: styles

File: src/routes/Dataset/components/DatasetChecks/CheckLicense.js [hunk 3]
```diff
@@ -1,35 +1,38 @@
-import React, { Component } from 'react'
+import React from 'react'
+import PropTypes from 'prop-types'
 
 import Check from './Check'
 
-import { ACCEPTED_LICENSES } from '../../../../helpers/dataGouvChecks'
+import { ACCEPTED_LICENSES } from 'common//helpers/dataGouvChecks'
```
Comment (tmerlier) [inline src/routes/Dataset/components/DatasetChecks/CheckLicense.js]: `//`

**Pull request merged**